### PR TITLE
Deal with [JAR...] notes, CSVs for tables, other writeup-related fixes

### PR DESCRIPTION
**Pull request by jar398**
- Provide CSV for all tables in the writeup
- Address most [JAR ...] notes
- Deleted section on extinct annotations.  Not sure it's so
    interesting now?  We can reinstate if necessary.
- Removed inconsistency case from merge rules.
    Could not find an example to fit the rule.
    We can reinstate if necessary with a different example.
- Changes to merge conflict logging to facilitate example-finding
- Fix bug in "x subgenus x" processing
- Suppress sibling_higher flag for incertae sedis children
- Misc minor changes to text
- Fix classification of certain merge events (not inconsistent)
- Makefile rule for ids_in_synthesis  (needed for writeup)
- Stub README file, to be fleshed out later
- Remove the Chromista spreadsheet; it wasn't helpful any more
- Debugging command-line script for showing lineage of a taxon

### CODE REVIEW

File: doc/method/results.md [hunk 1]
```diff
@@ -1,35 +1,9 @@
 
-## Contributions of source taxonomies to the assembly
```
Comment (kcranston) [inline doc/method/results.md]: why did you remove this table?

Comment (jar398) [inline doc/method/results.md]: It's moved, not removed.  Scripted generation is now part of document build process.
